build_tools/waf_dynamo.py

```diff
@@ -502,6 +504,11 @@ def default_flags(self):
 
     platform_setup_vars(self, build_util)
 
+    hostfs = ''
+    if 'DM_HOSTFS' in self.env:
+        hostfs = self.env['DM_HOSTFS']
+    self.env.append_unique('DEFINES', 'DM_HOSTFS=\"%s\"' % hostfs)
```

Comment:
This is used in unit tests to allow for reading data on the host PC, when running from a device.